persistence/persistence-mybatis/src/main/resources/org/cbioportal/persistence/mybatis/MutationMapper.xml

```diff
@@ -201,6 +202,31 @@
             LIMIT #{limit} OFFSET #{offset}
         </if>
     </select>
+    
+    <select id="getMutationsInMultipleMolecularProfilesByAnnotation" resultType="org.cbioportal.model.Mutation">
+        SELECT
+        <include refid="select"/>
+        <include refid="from"/>
+        <if test="annotation != null">
+          SELECT * FROM mutation WHERE mutation.annotation = #{annotation}
```

Comment:
This is incorrect. You already have `SELECT` in this statement. `mutation.annotation = #{annotation}` should be included in the WHERE clause which is in `whereInMultipleMolecularProfiles `